### DISCUSSION TITLE
hide measure tool

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/itwinjs-core#7325] [Q&A] [answered]
Hello good morning everyone
on our webapp we are using @itwin/measure-tools-react": "^0.23.1" , our client would like if possible to eliminate the measure tool , which according to him replicates the functionality
I attach an image:
<img width="207" alt="measure tool" src="https://github.com/user-attachments/assets/0bbf26ff-2eae-48b6-9493-1801bfee334b">

I tried to use defaultUiConfig with hideToolSettings: true to try to disable the functionality but without success

how can I solve it

thanks Paul

**Comment by aruniverse** [comment, marked as answer]
Morning! Have you tried removing the measure tools ui provider?

**Reply by Paolo-lab** [reply]
Thanks @aruniverse

sometimes I get lost in a glass of water

Thanks again
